test/message/promise_fast_reject.js

```diff
@@ -0,0 +1,10 @@
+'use strict';
```

Comment:
Oh, cool - so we're specifying that in this PR the oldest error always gets reported to the stack trace. Might want to mention this in the test or test name :)